src/main/java/org/eclipse/golo/compiler/CodeGenerationResult.java

```diff
@@ -27,7 +29,11 @@
    * @param packageAndClass the package and class descriptor for the bytecode.
    */
   public CodeGenerationResult(byte[] bytecode, PackageAndClass packageAndClass) {
-    this.bytecode = bytecode;
+    if (bytecode == null) {
```

Comment:
Again, this is all compiler-internal, and I'm not sure it helps. But since it is not performance-criticial compared to the runtime I would say yes for this one :smile: